src/workforce-management/Views/Employee/Detail.cshtml

```diff
@@ -0,0 +1 @@
+﻿<h4>HELLO!</h4>

```

Comment:
delete this if it's not part of your pull request, this isn't technically working code imho